docker-compose.yml

```diff
@@ -1,18 +1,22 @@
 web:
   build: .
+  dockerfile: Dockerfile.development
   volumes:
     - .:/champaign
     - ../sou-assets:/sou-assets
   ports:
     - "3000:3000"
+    - "8080:8080"
   links:
     - db
     - redis
   environment:
     PG_USERNAME: postgres
     PG_HOST: db
+    PG_PORT: 5432
+
   env_file:
-    - env.yml
+    - env-docker.yml
```

Comment:
Anyone using this branch will need to create a separate `env-docker.yml` file (simply copy the `env.yml` file you've got and remove `EXTERNAL_JS_PATH` or `BRAINTREE_TOKEN_URL`. If you don't already have an `env.yml` file, create one copying the template file provided (`env.yml.template`).